README.md

```diff
@@ -1,9 +1,25 @@
 # Support
 
 Forms that create Zendesk tickets for requests coming from government agencies.
-The app is written in Rails and uses the zendesk_api gem to connect to Zendesk.
 
-### Getting started
+## Technical documentation
+
+This is a Ruby on Rails application which is not public facing. However, it does face GOV.UK employees and is used as a way to submit Zendesk tickets. 
+The Zendesk tickets are created either from the information submitted in the `support` app forms or via the information collected from public 
+users which is forwarded from the `feedback` app. The `feedback` app is public facing and serves its own contact forms.
+
+The app makes use of the `gds_zendesk` gem to connect to Zendesk. 
+
```

Comment:
We should mention the feedback explorer which allows users to browse the feedback submitted from GOV.UK pages, and to export CSVs of the same for further analysis.

I think it might also be worth de-emphasising zendesk as it raises the question "what's this app for? why not just use zendesk directly?".  The support app lets people who use the GOV.UK publishing tools ask for specific help (creating a new campaign site, making content changes, asking for user permissions, asking for feature requests, etc...) from the GOV.UK team.   It may be worth listing all these types, or pointing at the place in the code where these request types are defined (if it's easy to interpret).